collections/_posts/2022-10-17-DiffusionModel.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+author: "Sophie Carneiro Esteves"
+cite:
+    authors: "Jascha Sohl-Dickstein, Eric A. Weiss, Niru Maheswaranathan and Surya Ganguli"
+    title:   "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+    venue:   "International Conference on Machine Learning 2015"
+pdf: "http://proceedings.mlr.press/v37/sohl-dickstein15.pdf"
+---
+
+# Notes
+- Code is available on GitHub: [https://github.com/Sohl-Dickstein/Diffusion-Probabilistic-Models](https://github.com/Sohl-Dickstein/Diffusion-Probabilistic-Models)
+
+# Introduction
+The authors propose a new probalistic model that aims to link a known distribution $$ \pi(.) $$ to another one $$ q(.) $$ based on diffusion. To achieve it, they iteratively added small amount of noise on the target distribution until it is converted into the known distribution. This process can be seen has a Markov chain that can be learned reversly. 
+
+
+# Highlights
+They proposed a new framework based on physics that allows: 
+- To generate data from a known distribution by defining thousands of steps that have simple functional form.
+- To tract changes through data generation with these steps.
+- To combine different distributions (with a conditional distribution, you can do inpainting)
```

Comment:
Typos + I'd be careful about making analogies to physics because of the name _diffusion_. I'd rather keep it simple and say it's an _iterative_ framework :stuck_out_tongue: 
```suggestion
They proposed a new iterative framework that allows: 
- To generate data from a known distribution by defining thousands of steps that have a simple functional form.
- To tract changes through data generation using these simple, incremental steps.
- To combine different distributions (e.g. with a conditional distribution, you can do inpainting)
```